muffnn/autoencoder/autoencoder.py

```diff
@@ -176,8 +177,7 @@ def _set_up_graph(self):
         second_layers \
             = list(self.hidden_units[::-1][1:]) + [self.input_layer_size_]
         for i, layer_sz in enumerate(second_layers):
-            if self.dropout is not None:
```

Comment:
it looks like [these objects](https://github.com/tensorflow/tensorflow/blob/r0.11/tensorflow/python/ops/nn_ops.py#L1187-L1196) still get added to the graph and are probably not trivial computationally even when 1.0 is fed in for `keep_prob`.